app/src/main/res/layout/fragment_quest_answer.xml

```diff
@@ -120,6 +120,58 @@
 
             </LinearLayout>
 
+            <androidx.core.widget.NestedScrollView
```

Comment:
As far as I can see, this nested scroll view makes no sense here, it should be removed.

Reply:
Resolved by fa183c21d